README

```diff
@@ -1,4 +0,0 @@
-Pub Standards Lancaster: The post-conference drinkup, without the conference.
```

Comment:
Can you leave the README around? I'll happily accept updates to it though, if you have something in mind.

Reply:
Of course.  I don't remember deleting it or why I would have done that!  So yeah, not a problem :)